test/test_mais.py

```diff
@@ -74,27 +76,17 @@ def test_get_response_success(access_token, base_url, client_id, client_secret):
 
 
 @pytest.mark.mais_tests
```

Comment:
We can remove these markers now since we don't need to skip them. Also the we don't need this anymore:

https://github.com/sul-dlss/rialto-airflow/blob/ab179f369dae5877fe497cadb329b5f3c2ee4575/pyproject.toml#L31

Reply:
removed